### DISCUSSION TITLE
JUST SUCCESSFULLY PUBLISHED!

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/DP2SceneGraphWizard#42] [General]
After several days stumbling, then finding DP2SceneGraphWizard & CFM, I just got my channel switched over to SDK and approved.🙏🏻  THANK YOU, Michael...and community!

In case it helps anyone...
SOFWARE VERSIONS:
DP2SGW version 1.0.0.0alpha 7.3 (or 7.2...can't quite remember - it's been a long few days on this)
CFM version 1.0.0.7
Note that I'm doing this on a Mac, using an instance of Windows via Amazon EC2 (first timer with that...just for these tools)
Contents in feed so far:
1 episode of 1 TV series 
About 25 shortFormVideos

**Comment by ferdiworks** [comment]
@adamesq Did you do the4 channels with Ads?

**Reply by adamesq** [reply]
Hi @ferdiworks - sorry for the delay! I've been keeping tabs on this, but haven't had a chance to go back and try ads yet. Hopefully in the next couple of days I will. I saw that @rrirower did provide a sample ad feed to try out.

**Comment by rrirower** [comment]
@adamesq please add your experiences to the "Success stories" discussion.

**Reply by adamesq** [reply]
YES! Absolutely! Done! I need to jump back in soon. I had to take off my content hat & put on the ⚖️ one for some IP offenders!
